test/unit/org/joda/time/base/AbstractIntervalTests.groovy

```diff
@@ -0,0 +1,56 @@
+package org.joda.time.base
+
+import au.org.emii.aatams.JodaOverrides
+import au.org.emii.aatams.OpenInterval
+
+import org.joda.time.DateTime
+import org.joda.time.Interval
+import org.joda.time.base.AbstractInterval
+
+import grails.test.*
+
+class AbstractIntervalTests extends GrailsUnitTestCase {
+
+    def now = new DateTime()
+
+    void setUp() {
+        super.setUp()
+        JodaOverrides.apply()
+    }
+
+    void tearDown() {
+        AbstractInterval.metaClass = null
+        super.tearDown()
+    }
+
+    void testContainsList() {
+        def interval = new Interval(
+            now,
+            now.plusDays(3)
+        )
+
+        def yesterday = now.minusDays(1)
+        def tomorrow = now.plusDays(1)
+        def nextWeek = now.plusWeeks(1)
+
+        assertTrue(interval.contains([ yesterday, tomorrow ]))
+        assertTrue(interval.contains([ tomorrow ]))
+        assertTrue(interval.contains([ now ]))
+        assertFalse(interval.contains([ yesterday, nextWeek ]))
+        assertTrue(interval.contains([ yesterday, tomorrow, nextWeek ]))
+
+        assertFalse(interval.contains([ ]))
+        assertFalse(interval.contains([ null ]))
+        assertFalse(interval.contains([ null, yesterday ]))
+        assertTrue(interval.contains([ null, tomorrow ]))
+    }
```

Comment:
I like variables named like this, I think they make tests easier to read. You might consider renaming `interval` to `nextThreeDays` (or whatever) so that it is "named" as well.